Scripts/Runtime/Entities/Tasks/ITaskSystem.cs

```diff
@@ -1,7 +1,17 @@
+using Unity.Entities;
+
 namespace Anvil.Unity.DOTS.Entities
 {
     public interface ITaskSystem
     {
-        
+        public byte Context
+        {
+            get;
+        }
+
+        public World World
+        {
+            get;
+        }
```

Comment:
🚲 you can inline these if you want. My style preference has always been.
``` csharp
public int AutoGet { get; }

public int OneLineGet
{
   get => GetMyInt();
}

public int MultiLineGet
{
   get
   {
      int i = GetMyInt()+1;
      return i;
   }
}
```

Reply:
Yes you are right. As I see these I'm editing to make those changes. I have done this specific file of course already.